tools/make.py

```diff
@@ -103,8 +103,11 @@
                       default=None,
                       help='For some commands you can use filter to filter out results')
 
-    parser.add_argument("--stats-depth", type=int, dest="stats_depth",
-                      default=2, help="Depth level for static memory report")
+    parser.add_argument("--stats-depth",
+                      type=int,
+                      dest="stats_depth",
+                      default=2,
+                      help="Depth level for static memory report")
```

Comment:
This, and the 3 prior lines, still look like they do not line up with the character after the `(`, which is the style we have throughout the tools.